src/interpolaters/Interpolate_transform.res

```diff
@@ -28,11 +28,31 @@ let interpolateTransform = (~range as (rl, rh), ~domain as (dl, dh), ~value) =>
     ((transforms |> Js.Array.joinWith(", ")) ++ ")"))
 }
 
+let populateTransformRegistry = transforms => {
+  transforms |> Js.Array.reduce((registry, t) => {
+    let property = Js.String.substring(~from=0, ~to_=Js.String.indexOf("(", t), t)
+
+    Js.Dict.set(registry, property, t)
+
+    registry
+  }, Js.Dict.empty())
+}
+
 let interpolateTransforms = (~range, ~domain as (dl, dh), ~value) => {
   let dlTransforms = Parse_transform.parseTransforms(dl)
   let dhTransforms = Parse_transform.parseTransforms(dh)
 
-  dlTransforms
-  |> Array.mapi((i, t) => interpolateTransform(~range, ~domain=(t, dhTransforms[i]), ~value))
+  let dlTransformRegistry = populateTransformRegistry(dlTransforms)
+  let dhTransfromRegistry = populateTransformRegistry(dhTransforms)
+
+  dlTransformRegistry
+  |> Js.Dict.entries
+  |> Js.Array.map(((property, t)) =>
+    interpolateTransform(
+      ~range,
+      ~domain=(t, Js.Dict.unsafeGet(dhTransfromRegistry, property)),
```

Comment:
The `unsafeGet` will go away when we add support for animatable "none". In this case, where the `to` value doesn't have a property that `from` has, or vice-versa, we'll patch it with an animatable "none".